tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_4_durability_and_cover/test_table_4_1.py

```diff
@@ -185,44 +108,6 @@ def test_description_of_the_environment(self) -> None:
         assert Chemical.NA.description_of_the_environment() == "Not applicable"
 
 
-class TestExposureClasses:
-    """Testing ExposureClasses class."""
-
-    def test_str(self) -> None:
-        """Check if the __str__ method returns the correct string representation of the exposure classes."""
-        exposure_classes = ExposureClasses(
-            carbonation=Carbonation("XC3"),
-            chloride=Chloride("XD2"),
-            chloride_seawater=ChlorideSeawater("Not applicable"),
-            freeze=FreezeThaw("XF2"),
-            chemical=Chemical("XA2"),
-        )
-        assert str(exposure_classes) == "XC3, XD2, XF2, XA2"
-
-    def test_str_x0(self) -> None:
-        """Check if the __str__ method returns the correct string representation of the exposure classes
-        when all exposure classes are not applicable.
-        """
-        exposureclasses = ExposureClasses()
-        assert str(exposureclasses) == "X0"
-
-    def test_no_risk(self) -> None:
-        """Check if the no_risk method returns True if the exposure classes are all not applicable."""
-        exposureclasses = ExposureClasses()
-        assert exposureclasses.no_risk is True
-
-    def test_no_risk_false(self) -> None:
-        """Check if the no_risk method returns False if at least one exposure class is applicable."""
-        exposureclasses = ExposureClasses(
-            carbonation=Carbonation("XC3"),
-            chloride=Chloride("XD2"),
-            chloride_seawater=ChlorideSeawater("Not applicable"),
-            freeze=FreezeThaw("XF2"),
-            chemical=Chemical("XA2"),
-        )
-        assert exposureclasses.no_risk is False
-
-
 def test_comparing_different_types_raises_error() -> None:
     """Check if comparing different exposure class types, raises TypeError."""
     with pytest.raises(TypeError):
```

Comment:
To solve the errors regarding 'Statement seems to have no effect'. You could add an underscore as a variable
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/1edc02ff-f466-4908-b793-b85ec5543977)

Reply:
Done!
I did not get these errors in my IDE though! 🤷‍♂️